src/main/java/org/jboss/forge/arquillian/command/algeron/AddAlgeronMavenRetriever.java

```diff
@@ -0,0 +1,88 @@
+package org.jboss.forge.arquillian.command.algeron;
+
+import org.jboss.forge.addon.dependencies.builder.DependencyBuilder;
+import org.jboss.forge.addon.ui.context.UIBuilder;
+import org.jboss.forge.addon.ui.context.UIContext;
+import org.jboss.forge.addon.ui.hints.InputType;
+import org.jboss.forge.addon.ui.input.UIInput;
+import org.jboss.forge.addon.ui.metadata.UICommandMetadata;
+import org.jboss.forge.addon.ui.metadata.WithAttributes;
+import org.jboss.forge.addon.ui.util.Categories;
+import org.jboss.forge.addon.ui.util.Metadata;
+
+import javax.inject.Inject;
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+public class AddAlgeronMavenRetriever extends AbstractAlgeronRetrieverCommand
+{
```

Comment:
I saw how everything was reformatted, so in theory it should. Maybe in this case I was doing it manually because this is what I did in the past.